client/lib/plans/constants.js

```diff
@@ -181,6 +181,14 @@ export const plansList = {
 			FEATURE_GOOGLE_ANALYTICS,
 			FEATURE_NO_BRANDING
 		] ),
+		getPromotedFeatures: () => [
```

Comment:
Oh, thanks for clarification. So, we actually need a list of globally promoted features for each plan. Let's keep it. :+1: